account_invoice_line_description/models/invoice.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Agile Business Group sagl
+# (<http://www.agilebg.com>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, models
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = "account.invoice.line"
+
+    @api.onchange('product_id')
+    def _onchange_product_id(self):
+        res = super(AccountInvoiceLine, self)._onchange_product_id()
+        part = self.invoice_id.partner_id
+        type = self.invoice_id.type
+        if self.product_id:
+            if part.lang:
+                product = self.product_id.with_context(lang=part.lang)
+            else:
+                product = self.product_id
+
+            if type in ('in_invoice', 'in_refund'):
+                if product.description_purchase and self.user_has_groups(
```

Comment:
Check as first thing the group and return if not belonging to to avoid all of these lines.